benchmarks/src/benchdummy.cpp

```diff
@@ -1,3 +1,6 @@
+// SPDX-FileCopyrightText: 2022 The Ikarus Developers mueller@ibb.uni-stuttgart.de

```

Comment:
May be we can move benchdummy to something in Ikarus Examples and remove this folder?

Reply:
Maybe we can simply delete it

Reply:
Sure. It isn't related to Ikarus much as well

Reply:
I deleted it